app/uk/gov/hmrc/bindingtariffclassification/scheduler/DaysElapsedJob.scala

```diff
@@ -39,9 +41,30 @@ class DaysElapsedJob @Inject()(appConfig: AppConfig, caseService: CaseService) e
 
   override def interval: FiniteDuration = FiniteDuration(jobConfig.intervalDays, TimeUnit.DAYS)
 
-  override def execute(): Future[String] = {
-    caseService.incrementDaysElapsedIfAppropriate(jobConfig.intervalDays, appConfig.clock)
-      .map(count => s"Incremented the Days Elapsed for [$count] cases.")
+  override def execute(): Future[Unit] = {
+    ifTodayIsAWorkingDay {
+      caseService.incrementDaysElapsed(jobConfig.intervalDays)
+        .map(count => Logger.info(s"Scheduled Job [$name]: Incremented the Days Elapsed for [$count] cases."))
+    }
+  }
+
+  private def ifTodayIsAWorkingDay(function: => Future[Unit]): Future[Unit] = {
+    val today = LocalDate.now(appConfig.clock)
+    val dayOfTheWeek = today.getDayOfWeek
+    if (dayOfTheWeek == DayOfWeek.SATURDAY || dayOfTheWeek == DayOfWeek.SUNDAY) {
+      Logger.info(s"Scheduled Job [$name]: Skipping today as it is a Weekend")
+      Future.successful()
```

Comment:
You need to add an extra `()` for the Unit
```
Future.successful(())
```

Reply:
Fixed